README.md

```diff
@@ -64,18 +64,40 @@ export EIP_CONFIG_PATH=$DISTRO_PATH/eip_config
 
 The complete list of available variables can be found in [.env](.env).
 
+### Move to the location of bahmni-docker-compose project
+
+```bash
+cd $BAHMNI_DIR/bahmni-docker-compose-$VERSION
+```
+
 ### Start Bahmni:
 
+<table>
+<tr>
+<td> macOS </td> <td> Linux </td>
+</tr>
+<tr>
+<td>
+
+```bash
+docker compose -p $DISTRO_GROUP up
 ```
-cd $BAHMNI_DIR/bahmni-docker-compose-$VERSION && \
-docker-compose -p $DISTRO_GROUP up
+
+</td>
+<td>
+
+```bash
+sudo -E docker compose -p $DISTRO_GROUP up
```

Comment:
Well... If correctly set up Docker should run without `sudo`.
In general, we never see instructions run with `sudo`. That's why the point in the **Important** section.

Maybe we can create a "Prerequisites" section of the README that tells 1. Which version of Docker to run, 2. That it assumes Docker to be runnable as standard users.

What do you think @kdaud ?

Reply:
> What do you think @kdaud ?

For normal users on Linux os the last installation step requires to prepend `sudo` or `sudo -E`  to have a successful run otherwise it throws exceptions of this [kind](https://talk.openmrs.org/t/ozone-installation/38770/4) and other [permission related](https://talk.openmrs.org/t/ozone-setup-error-via-docker/38800) logs.

> Maybe we can create a "Prerequisites" section of the README that tells 1. Which version of Docker to run, 2. That it assumes Docker to be runnable as standard users.

After testing out different versions of docker, I noticed all docker versions >20.10.12 run well including the latest SNAPSHOT so I would think we should not restrict a particular version for users, may be we can  recommend the minimum version to be used.

Reply:
> For normal users on Linux os the last installation step requires to prepend sudo or sudo -E to have a successful run otherwise it throws exceptions of this [kind](https://talk.openmrs.org/t/ozone-installation/38770/4) and other [permission related](https://talk.openmrs.org/t/ozone-setup-error-via-docker/38800) logs.

I don't think that's necessary true.
1. "Normal users" on Linux OS should have followed the complete installation options, which includes adding the current user to a docker group and with sudo access.

2. The first link you paste I think is actually due to an incorrect Docker Compose version. In older versions of Docker Compose, the expected format of the values of some of the keys have changed format (single value VS array for instance)
3. The second link you've shard is actually because running Maven as `sudo` (root). Don't know why this person was runnning Maven as sudo...

> After testing out different versions of docker, I noticed all docker versions >20.10.12 run well including the latest SNAPSHOT so I would think we should not restrict a particular version for users, may be we can recommend the minimum version to be used.

OK fair enough. I actually had rather in mind to advise the version of Docker **Compose**, not Docker itself.
That's because of the issue explained above. Plus, in older version you would run `docker-compose`. In newer version, `docker compose`